honeybee/cli/edit.py

```diff
@@ -0,0 +1,292 @@
+"""honeybee model editing commands."""
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click is not installed. Try `pip install . [cli]` command.'
+    )
+
+from ladybug_geometry.geometry3d.pointvector import Vector3D
+
+from honeybee.model import Model
+from honeybee.room import Room
+from honeybee.facetype import face_types, Wall
+from honeybee.boundarycondition import Outdoors
+from honeybee.boundarycondition import boundary_conditions as bcs
+try:
+    ad_bc = bcs.adiabatic
+except AttributeError:  # honeybee_energy is not loaded and adiabatic does not exist
+    ad_bc = None
+
+
+import sys
+import os
+import logging
+import json
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands for editing Honeybee models.')
+def edit():
+    pass
+
+
+@edit.command('convert-to-units')
+@click.argument('model-json', type=click.Path(
+    exists=True, file_okay=True, dir_okay=False, resolve_path=True))
+@click.argument('units', type=str)
+@click.option('--output-file', help='Optional file to output the Model JSON string'
```

Comment:
Can we have the option just to change the units and not re-size the geometry? Like how Rhino asks if one wants to re-size the model when the units are changed.

Reply:
Yes, we can add a flag for that option.

Reply:
Added!